environment.yaml

```diff
@@ -0,0 +1,122 @@
+  - intel-openmp=2023.0.0=h57928b3_25922
+  - libblas=3.9.0=16_win64_mkl
+  - libcblas=3.9.0=16_win64_mkl
+  - libffi=3.4.2=h8ffe710_5
+  - libhwloc=2.9.0=h51c2c0f_0
+  - libiconv=1.17=h8ffe710_0
+  - liblapack=3.9.0=16_win64_mkl
+  - libsqlite=3.40.0=hcfcfb64_0
+  - libxml2=2.10.3=hc3477c8_6
+  - libzlib=1.2.13=hcfcfb64_4
+  - mkl=2022.1.0=h6a75c08_874
+  - openssl=3.1.0=hcfcfb64_0
+  - pip=23.0.1=pyhd8ed1ab_0
+  - pthreads-win32=2.9.1=hfa6e2cd_3
+  - pydensecrf=1.0rc3=py310h1c4a608_4
+  - python=3.10.10=h4de0772_0_cpython
+  - python_abi=3.10=3_cp310
+  - setuptools=67.6.1=pyhd8ed1ab_0
+  - tbb=2021.8.0=h91493d7_0
+  - tk=8.6.12=h8ffe710_0
+  - ucrt=10.0.22621.0=h57928b3_0
+  - vc=14.3=hb6edc58_10
+  - vs2015_runtime=14.34.31931=h4c5c07a_10
+  - wheel=0.40.0=pyhd8ed1ab_0
+  - xz=5.2.6=h8d14728_0
+  - pip:
+      - aiohttp==3.8.4
+      - aiosignal==1.3.1
+      - antlr4-python3-runtime==4.9.3
+      - asn1crypto==1.5.1
+      - async-timeout==4.0.2
+      - attrs==22.2.0
+      - backports-cached-property==1.0.2
+      - certifi==2022.12.7
+      - chardet==3.0.4
+      - charset-normalizer==2.1.1
+      - colorama==0.4.6
+      - coloredlogs==15.0.1
+      - ctranslate2==3.11.0
+      - deepl==1.14.0
+      - editdistance==0.6.2
+      - einops==0.6.0
+      - filelock==3.9.0
+      - flatbuffers==23.3.3
+      - freetype-py==2.3.0
+      - frozenlist==1.3.3
+      - ftfy==6.1.1
+      - googletrans==4.0.0rc1
+      - h11==0.9.0
+      - h2==3.2.0
+      - hpack==3.0.0
+      - hstspreload==2023.1.1
+      - httpcore==0.9.1
+      - httpx==0.13.3
+      - huggingface-hub==0.13.4
+      - humanfriendly==10.0
+      - hyperframe==5.2.0
+      - idna==2.10
+      - imagehash==4.3.1
+      - imageio==2.27.0
+      - jinja2==3.1.2
+      - kornia==0.6.11
+      - langid==1.1.6
+      - lazy-loader==0.2
+      - markupsafe==2.1.2
+      - marshmallow==3.19.0
+      - mpmath==1.2.1
+      - multidict==6.0.4
+      - nest-asyncio==1.5.6
+      - networkx==3.0
+      - numpy==1.24.1
+      - omegaconf==2.3.0
+      - onnxruntime==1.14.1
+      - open-clip-torch==2.16.0
+      - openai==0.27.4
+      - opencv-python==4.7.0.72
+      - oscrypto==1.3.0
+      - packaging==23.0
+      - pandas==2.0.0
+      - pillow==9.3.0
+      - protobuf==3.20.3
+      - pyclipper==1.3.0.post4
+      - pyreadline3==3.4.1
+      - python-dateutil==2.8.2
+      - python-dotenv==1.0.0
+      - pytz==2023.3
+      - pywavelets==1.4.1
+      - pyyaml==6.0
+      - regex==2023.3.23
+      - requests==2.28.1
+      - rfc3986==1.5.0
+      - safetensors==0.3.0
+      - scikit-image==0.20.0
+      - scipy==1.10.1
+      - sentencepiece==0.1.97
+      - shapely==2.0.1
+      - six==1.16.0
+      - sniffio==1.3.0
+      - sympy==1.11.1
+      - tensorboardx==2.6
+      - tifffile==2023.3.21
+      - timm==0.6.13
+      - tokenizers==0.13.3
+      - torch==2.0.0+cu118
+      - torch-summary==1.4.5
+      - torchaudio==2.0.1+cu118
+      - torchvision==0.15.1+cu118

```

Comment:
The environment requires torch with CUDA enabled, which is not required by this project.